staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1040,7 +1047,9 @@ func filterWithAttrsFunction(key string, p storage.SelectionPredicate) filterWit
 
 // LastSyncResourceVersion returns resource version to which the underlying cache is synced.
 func (c *Cacher) LastSyncResourceVersion() (uint64, error) {
-	c.ready.wait()
+	if err := c.ready.wait(); err != nil {
+		return 0, errors.NewServiceUnavailable(err.Error())
+	}
```

Comment:
where is this used? The other call sites look clearer, representing CRUD operation. Not this one.

Reply:
IIRC it was used historically but now it is only used in tests I think.